CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Unreleased
 ----------
 
 ### Added
+-   Adds French phoneme list and filtered tsv file. (\#213)
```

Comment:
"TSV"

Reply:
Okay, I think it should be taken care of.